src/webview/handleOutboundEvents.js

```diff
@@ -55,8 +55,8 @@ type WebViewOutboundEventScroll = {|
   endMessageId: number,
 |};
 
-type WebViewOutboundEventAvatar = {|
-  type: 'avatar',
+type WebViewOutboundEventRequestProfile = {|
```

Comment:
Since the string is 'request-user-profile', I would probably name this Flow type alias `WebViewOutboundEventRequestUserProfile`. It's pretty hilariously long, but at least it's less ambiguous that they're meant to go together.